examples/pipelines/filter-cc-25m/pipeline.py

```diff
@@ -0,0 +1,64 @@
+# Pipeline code goes here
+from pathlib import Path
+import logging
+import sys
+
+sys.path.append("../")
+from fondant.pipeline import ComponentOp, Pipeline
+
+
+def create_directory_if_not_exists(path):
+    p_base_path = Path(path).resolve()
+    p_base_path.mkdir(parents=True, exist_ok=True)
+    cache_dir = p_base_path / PIPELINE_NAME / "cache"
+    cache_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
Temporary workaround to make sure that the example pipeline is working in a clean environment.
See issue: #440